inactivity.md

```diff
@@ -0,0 +1,36 @@
+---
+layout: default
+title: Maintainer and Repository Inactivity
+nav_order: 6
+---
+[//]: # (SPDX-License-Identifier: CC-BY-4.0)
+
+# Inactivity Policies
+
+As part of the normal lifecycle of a project codebases and maintainers come and go.  This document formalizes a Hyperledger wide polciy for archiving projects and moving maintainers to emereitus status when a proejct does not have an explicit policy of their own.
+
+## Maintainer Inactivity
+
+Hyperledger very much appreciates the contributions of all maintainers but removing write privledges is in the interest of an orderly and secure project.
+
+Maintainers who have not contributed to a Hyperledger project for (`exact timeframe to be decided`) 3 / 4 / 6 months will be removed from the relevant Github Organizations.
+
+Activity can be code contributions, code reviews, issue reporting, or any other such activity trackable by github attriubuted to a Hyperledger repository.
+
+Maintainers will notified of this via an "at" @ mention in a pull request moving their name from a projects active contributors to the emeritus contributors.
+
+Maintainers who express a desire to continue contributing will, at the maintainers request, be given a 3 month extension to contribute to the project.
+
+Maintainers who have been moved to emeritus status may be re-added by the projects when their contributions resume and the existing maintainers approve their reactivation.
+
+These policies apply to projects that do not have an expicity maintainer inactivity policy. Where a project has an established and functioning policy the projects policies shall prevail.
+
+## Repository Inactivity
```

Comment:
There is a separate process for project's lifecycle. We can focus on the maintainers in this section, if we need a revision on project's lifecycle let's do it there.

Reply:
I would prefer to keep all inactivity policies in one place.  Splitting it into two policies splits the focus, which is dealing with inactivity holistically and not piecemeal.

Reply:
It is possible that repositories for an otherwise active project have become inactive. It is important to have some mechanism to "archive" those inactive repositories. I agree that any project lifecycle changes should be in the project lifecycle document.